virtualenv/builders/base.py

```diff
@@ -44,7 +50,35 @@ def __init__(self, python, flavor, system_site_packages=False,
     def check_available(cls, python):
         raise NotImplementedError
 
+    def _get_base_python_bin(self):
```

Comment:
This seems to be called multiple times. You should cache the result as a subprocess call is a huge overhead.